src/app/api/simulations/request-powerpoint/route.ts

```diff
@@ -0,0 +1,27 @@
+import { NextResponse } from 'next/server'
+import { auth } from '~/lib/auth/auth'
+
+export async function POST(request: Request) {
+  const body = await request.json()
+  const session = await auth()
```

Comment:
This will need to be changed, since I rollbacked to next 4, we will need to use `await getServerSession(authOptions)`